esmf_regrid/experimental/unstructured_scheme.py

```diff
@@ -88,10 +88,6 @@ def _create_cube(data, src_cube, mesh_dim, grid_x, grid_y):
 
```

Comment:
See Line 85: describes a `mesh` object, but this is no longer part of the `_create_cube` signature